app/forms/declaration/Seal.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms.declaration
+import play.api.data.Forms
+import play.api.data.Forms.text
+import play.api.libs.json.Json
+import utils.validators.forms.FieldValidator.{isAlphanumeric, noLongerThan, nonEmpty}
+
+case class Seal(id: String)
+
+object Seal  {
+  implicit val format = Json.format[Seal]
+
+  val formId = "Seal"
+
+  val formMapping = Forms.mapping(
+    "id" ->
+      text()
+        .verifying("standard.transport.sealId.empty.error", nonEmpty)
+        .verifying("standard.transport.sealId.alphaNumeric.error", isAlphanumeric)
```

Comment:
Please change to `isEmpty or isAlphanumeric` because when input is empty we shouldn't show this error

Reply:
this error will not be shown id nonEmpty, I have test to prove